app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_cat_age_and_dog_age_equal_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_get_cat_age_and_dog_age_equal_zero2() -> None:
+    assert get_human_age(14, 14) == [0, 0]
```

Comment:
This test case `test_get_cat_age_and_dog_age_equal_zero2` is redundant as it tests the same scenario as `test_get_cat_age_and_dog_age_equal_zero`. Consider removing this duplicate test case to adhere to the 'Don't Repeat Yourself' principle from the checklist.